typing/typemod.ml

```diff
@@ -1980,7 +1984,8 @@ and type_module_aux ~alias sttn funct_body anchor env smod =
                                     Cannot_eliminate_dependency mty_functor))
                 in
                 begin match
-                  Includemod.modtypes ~loc:smod.pmod_loc env mty_res nondep_mty
+                  Includemod.modtypes ~mark:Mark_neither
+                    ~loc:smod.pmod_loc env mty_res nondep_mty
```

Comment:
If I understand correctly, this part is essentially an
```
assert( Includemod.modtypes ~mark:Mark_neither ~loc:smod.pmod_loc env mty_res nondep_mty = Tcoerce_None )
```
?

Reply:
That plus catching `Includemod.Error` exceptions coming out of `Includemod.modtypes`. Note that I'm only changing the `mark` parameter here.